### PR TITLE
[ISSUE #1078]fix User can't use mqadmin command normally if they don't copy the tool.yml file to related fold and AclEnable flag is closed. 

### PR DESCRIPTION
**Pull request by zongtanghu**
## What is the purpose of the change

(1)fix User can't use mqadmin command normally if they don't copy the tool.yml file to related fold and AclEnable flag is closed. 
## Brief changelog

(1)just add the block code:try and catch.

## Verifying this change

I have already verified this change in my local enviroment.

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: tools/src/main/java/org/apache/rocketmq/tools/command/MQAdminStartup.java [hunk 1]
```diff
@@ -249,8 +249,13 @@ public static void initCommand(SubCommand command) {
     public static RPCHook getAclRPCHook() {
         String fileHome = System.getProperty(MixAll.ROCKETMQ_HOME_PROPERTY, System.getenv(MixAll.ROCKETMQ_HOME_ENV));
         String fileName = "/conf/tools.yml";
-        JSONObject yamlDataObject = AclUtils.getYamlDataObject(fileHome + fileName ,
+        JSONObject yamlDataObject = null;
+        try {
+            yamlDataObject = AclUtils.getYamlDataObject(fileHome + fileName,
                 JSONObject.class);
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (dongeforever) [inline tools/src/main/java/org/apache/rocketmq/tools/command/MQAdminStartup.java]: Do not printStackTrace，just ignored

Comment (dongeforever) [reply tools/src/main/java/org/apache/rocketmq/tools/command/MQAdminStartup.java]: Separate different Exception will be a better choice.

File: acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java [hunk 2]
```diff
@@ -125,13 +125,12 @@ public static boolean isMinus(String minus) {
             fis = new FileInputStream(new File(path));
             return ymal.loadAs(fis, clazz);
         } catch (Exception e) {
-            throw new AclException(String.format("The  file for Plain mode was not found , paths %s", path), e);
```
Comment (dongeforever) [inline acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java]: Only ignore the FileNotFoundException

Comment (zongtanghu) [reply acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java]: Okay,I will adjust my codes in next commit.

**Comment by zongtanghu** [comment]
The unit test of DefaultMQPushConsumerTest cause the CI failed!From the log content,it hangs for some reasom.

File: acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java (hunk 2 above)
Comment (dongeforever) [reply acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java]: Great！

**Pull request merged**